packages/a11y-testing/src/facades/withProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import { Telemetry, Unstable_FluentContextProvider, ProviderContextPrepared } from '@fluentui/react-bindings';
+import { Renderer, noopRenderer } from '@fluentui/react-northstar-styles-renderer';
+import { emptyTheme, ThemePrepared } from '@fluentui/styles';
+import { mount, MountRendererProps, ComponentType } from 'enzyme';
+import * as React from 'react';
+
+export const EmptyThemeProvider: React.FunctionComponent<{
```

Comment:
@kolaps33 can we try to test if everything will work if we just don't wrap the component with theme provider. This was initially implemented like this, because the components will throw an error, but I am not sure what is the status at this moment. Would be great if we can remove the styles and other dependencies from this package.

Reply:
removed whole file and replace it with one line in `ComponentTestFacade.tsx`